scripts/run.sh

```diff
@@ -1,16 +1,160 @@
 #!/bin/bash
 
+[ "$DB_NAME" ] || DB_NAME='wordpress'
+[ "$DB_PASS" ] || DB_PASS='root'
+[ "$SEARCH_REPLACE" ] && \
+	BEFORE_URL=$(echo "$SEARCH_REPLACE" | cut -d ',' -f 1) && \
+	AFTER_URL=$(echo "$SEARCH_REPLACE" | cut -d ',' -f 2) \
+|| SEARCH_REPLACE=false
+
+# Error handling
+ERROR () { echo -e "\n=> $(tput setaf 1)$(tput bold)ERROR$(tput sgr 0) (Line $1): $2."; exit 1; }
+
+# Configure wp-cli
+if [ ! -f /app/wp-cli.yml ]; then
+	cat > /app/wp-cli.yml <<-EOF
+	quiet: true
+	apache_modules:
+	  - mod_rewrite
+
+	core config:
+	  dbuser: root
+	  dbpass: $DB_PASS
+	  dbname: $DB_NAME
+	  dbhost: db:3306
+
+	core install:
+	  url: $([ "$AFTER_URL" ] && echo "$AFTER_URL" || echo localhost:8080)
+	  title: $DB_NAME
+	  admin_user: root
+	  admin_password: $DB_PASS
+	  admin_email: admin@${DB_NAME}.com
```

Comment:
I mean the first time your run `docker-compose up`, it will generate the `wp-cli.yml` file with it. I didn't mean something dynamically updating.